frontend/src/utils/geojson.ts

```diff
@@ -74,6 +78,13 @@ function toClosestProviderToString(adequacies: Adequacies, method: Method, point
   }
 }
 
+function toClosestProviderDirection(adequacies: Adequacies, pointId: number, pointLat: number, pointLng: number) {
```

Comment:
seems like this could be a bit cleaner if it didn't have to know about all of the adequacies and the point id. Could just pass it one adequacy | null?

Reply:
Tried a few things but cant really get to something better. It's quite similar to the function right above it